src/ResourceStateBuilder.js

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import shortid from 'shortid'
+
+const rdf = require('rdf-ext')
```

Comment:
Is `rdf-ext` one of the libraries that doesn't function when loaded via `import rdf from 'rdf-ext'`?

Reply:
Not sure. Just cribbed this from elsewhere in the codebase.